src/main/java/com/epam/learn/assign1/model/Ticket.java

```
@@ -0,0 +1,25 @@
+package com.epam.learn.assign1.model;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+/**
+ * Created by maksym_govorischev.
+ */
+@Data
+@Accessors(chain = true)
+public class Ticket {
+    public enum Category {STANDARD, PREMIUM, BAR}
+
+    /**
+     * com.epam.learn.assign1.model.Ticket Id. UNIQUE.
+     * @return com.epam.learn.assign1.model.Ticket Id.
+     */
+    private Long id;
+    private Event event;
+    private User user;
+    private Category category;
+    private int place;
```

for model object I always prefer to use Object (like Integer or Boolean) and not primitive types, because in case you forgot to initialize them you will get default value (like zero or false), but if you use Integer or Boolean you will get null and potentially NPE, so you most probably "fail fast"